ocp_resources/user_defined_network.py

```diff
@@ -69,6 +69,19 @@ def to_dict(self) -> None:
     # End of generated code
 
 
+class TopologyType:
```

Comment:
I don't see any reason to have this class.
Each class should hold its constant. (like it was in `Layer2UserDefinedNetwork` class)

Reply:
Right, done.